packages/langchain/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./dist",
+    "tsBuildInfoFile": "./dist/.tsbuildinfo",
+    "composite": true
+  },
+  "include": ["src/**/*.ts"],
+  "exclude": ["dist/**/*", "test/**/*", "**/*.test.ts", "node_modules/**/*"],
+  "references": [{ "path": "../foundation-models" }, { "path": "../ai-api" }]
```

Comment:
[q] Is this needed?

Reply:
I would argue yes, because this package depends on the referenced packages, this ensures they are always aligned.